src/config/styleConfig.js

```diff
@@ -2,13 +2,12 @@
 import { Style, Circle, Stroke, Fill } from 'ol/style';
 
 // Convert '0.0' to '0'
-const cleanFloor = floor => {
-  return !isNaN(floor) ? parseFloat(floor, 10).toString() : floor;
-};
+const cleanFloor = (floor) =>
+  !isNaN(floor) ? parseFloat(floor, 10).toString() : floor;
 
-const lineStyler = lineStyle => {
-  return lineStyle.map(
```

Comment:
is Prettier or eslint that have done that ? I prefer to keep the return . Maybe it's done in the next commit.

Reply:
No idea, but prettier can handle both variants